tests/backup/backup_multiple_provisioner_test.go

```diff
@@ -222,3 +222,410 @@ var _ = Describe("{MultipleProvisionerCsiSnapshotDeleteBackupAndRestore}", Label
+				ctx, err := backup.GetAdminCtxFromSecret()
+				log.FailOnError(err, "Fetching px-central-admin ctx")
+				namespaceMappingkdmp := make(map[string]string)
+				for _, appCtx := range scheduledAppContextsForDefaultVscBackup {
+					namespaceMappingkdmp[appCtx.ScheduleOptions.Namespace] = appCtx.ScheduleOptions.Namespace + "-mul-app-kdmp"
+				}
+				restoreName := fmt.Sprintf("%s-%s-%s", "test-restore", "kdmp", RandomString(randomStringLength))
+				log.InfoD("Restoring namespaces from the [%s] backup", forceKdmpSchBackupName)
+				err = CreateRestoreWithValidation(ctx, restoreName, forceKdmpSchBackupName, namespaceMappingkdmp, make(map[string]string), DestinationClusterName, BackupOrgID, scheduledAppContextsForDefaultVscBackup)
+				dash.VerifyFatal(err, nil, fmt.Sprintf("Creating restore [%s] from backup [%s]", restoreName, forceKdmpSchBackupName))
+			}
+		})
+		Step("Restoring from the manual back backup", func() {
+			if len(backupNames) > 0 {
+				log.InfoD("Restoring from the manual back backup")
+				ctx, err := backup.GetAdminCtxFromSecret()
+				log.FailOnError(err, "Fetching px-central-admin ctx")
+				for i, appCtx := range scheduledAppContextsForCustomVscBackup {
+					scheduledNamespace := appCtx.ScheduleOptions.Namespace
+					restoreName := fmt.Sprintf("%s-%s-%s", "test-restore-manual-backup", scheduledNamespace, RandomString(randomStringLength))
+					log.InfoD("Restoring [%s] namespace from the [%s] backup", scheduledNamespace, backupNames[i])
+					err = CreateRestoreWithValidation(ctx, restoreName, backupNames[i], make(map[string]string), make(map[string]string), DestinationClusterName, BackupOrgID, scheduledAppContextsForCustomVscBackup[i:i+1])
+					dash.VerifyFatal(err, nil, fmt.Sprintf("Creation and Validation of restore [%s] from backup [%s]", restoreName, backupNames[i]))
+				}
+			}
+		})
+	})
+	JustAfterEach(func() {
+		if !Contains(runOnClusterProviders, GetClusterProvider()) {
+			// This test is meant to run only on IBM and OpenShift; later, it will be enabled on other configurations.
+			log.Infof("Skipping the test.This test is currently configured to run on IBM and openshift and environments. Future iterations will enable on other configs")
+			Skip("Skipping the test.This test is currently configured to run on IBM and openshift environments. Future iterations will enable on other configs")
+		}
+		defer EndPxBackupTorpedoTest(allAppContext)
+		defer func() {
+			log.InfoD("switching to default context")
+			err := SetClusterContext("")
+			log.FailOnError(err, "failed to SetClusterContext to default cluster")
+		}()
+
+		ctx, err := backup.GetAdminCtxFromSecret()
+		for _, scheduleName := range scheduleNames {
+			err = DeleteSchedule(scheduleName, SourceClusterName, BackupOrgID, ctx)
+			dash.VerifySafely(err, nil, fmt.Sprintf("Verification of deleting backup schedule - %s", scheduleName))
+		}
+
+		log.FailOnError(err, "Fetching px-central-admin ctx")
```

Comment:
need to move the verification to line 617

Reply:
done